Makefile

```diff
@@ -279,9 +279,9 @@ run-android: export TARGET := android
 # https://github.com/status-im/status-mobile/issues/18493
 run-android: export ORG_GRADLE_PROJECT_hermesEnabled := false
 run-android: ##@run Build Android APK and start it on the device
-	@scripts/run-android.sh
+	npx react-native run-android --appIdSuffix debug
```

Comment:
can you revert this? if you require this change it's better to do in an isolated pr 👍

Reply:
@J-Son89 that's temporary, will revert before merging